docs/developers/torch_add_a_new_converter.md

```diff
@@ -1,13 +1,37 @@
-# How To Add a New Torch Operator Converter(Deprecated soon)
+# How To Support a New Torch Operator
```

Comment:
maybe **Guide to add a new conversion from Torch op to Mhlo op** is more accurate ？

Reply:
It's not only developer guidance to Torch-To-Mhlo but an end-to-end developing introduction for a new Torch Operator.